### PR TITLE
Add features in the tiler config

### PR DESCRIPTION
**Pull request by Rub21**
@geohacker

### CODE REVIEW

**Comment by geohacker** [comment]
@Rub21 before we add these, can you check if these features are not already in the `other_points` table? https://github.com/developmentseed/osm-seed/blob/f68c4d48f4308ee027e55d00bc971a3770c6e0d1/images/tiler-imposm/config/imposm3.json#L570

**Comment by Rub21** [comment]
@geohacker  yes,  I checked that , those items not exist in the config neither in the DB

File: images/planet-dump/start.sh [hunk 1]
```diff
@@ -30,7 +30,8 @@ osmosis --read-apidb \
 # AWS
 if [ $CLOUDPROVIDER == "aws" ]; then
 	# Save the path file
-	echo "https://$AWS_S3_BUCKET.s3.amazonaws.com/planet/$planetPBFFile" > $stateFile
+	AWS_URL=${AWS_S3_BUCKET/s3:\/\//http:\/\/}
+	echo "$AWS_URL.s3.amazonaws.com/planet/$planetPBFFile" > $stateFile
```
Comment (geohacker) [inline images/planet-dump/start.sh:34]: @Rub21 any reason why we switched from s3 url to http? Also why not https?

Comment (Rub21) [reply images/planet-dump/start.sh:34]: That was ,because Dan , asked to change  that to http.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,194 +1,51 @@
 version: '3'
 services:
-  #   web:
```
Comment (geohacker) [inline docker-compose.yml:3]: Let's add these back

**Comment by geohacker** [comment]
May be we should merge https://github.com/developmentseed/osm-seed/pull/186/files before this PR.

**Pull request merged**